nodejs/week1/meal_sharing_website/src/backend/routes/meals.js

```diff
@@ -0,0 +1,35 @@
+//Respond with the json for all the meals.
+//For each meal, if there are reviews matching it's meal ID, add these reviews to each meal in the form of an array. 
+//For meals that do not have any reviews, the "reviews" property will be an empty array.
```

Comment:
I like this :) I find this very helpful to remind yourself what you need to implement.

Reply:
(you wouldn't have it in a real application, but it is good to have it for yourself)